python/pyspark/ml/regression.py

```diff
@@ -609,6 +611,147 @@ class GBTRegressionModel(TreeEnsembleModels):
     """
 
 
+@inherit_doc
+class AFTSurvivalRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol,
+                            HasFitIntercept, HasMaxIter, HasTol):
+    """
+    Accelerated Failure Time(AFT) Model Survival Regression
+
+    Fit a parametric AFT survival regression model based on the Weibull distribution
+    of the survival time.
+    see also, `https://en.wikipedia.org/wiki/Accelerated_failure_time_model`
```

Comment:
use Sphinx `seealso` http://sphinx-doc.org/markup/para.html#directive-seealso Please generate the HTML doc and test the link.